core/src/test/java/org/bitcoinj/script/ScriptChunkSizeTest.java

```diff
@@ -0,0 +1,65 @@
+package org.bitcoinj.script;
```

Comment:
This new class is missing the license. Also, while you're at it, can you make it that the last line is terminated with an EOL like all the others?

Reply:
done